config/locales/en.yml

```diff
@@ -62,7 +62,7 @@ en:
           something_else: Edit something else on the landing page
         subtopic_edit:
           accordions: "Edit %{page_name} accordions"
-          something: Edit something else on the
+          something: Edit something else on the %{page_name}
```

Comment:
Great catch on this missing variable 👍 

It seems a curious choice of the original implementer to call this `something` rather than `something_else` as that means the opposite.

Reply:
Oh and I just notice two lines above it's called `something_else` in a different context. 

I have an awkward feeling I may have just missed this in a review. Any chance you can fix? 🙏